portifolio.html

```diff
@@ -64,5 +64,82 @@ <h2>Software Developer</h2>
         <li><img src="image/Icon-Medium.png" alt="M" class="media"></li>
     </ul>
 </div>
+<div class="arrow">
+    <img src="image/arrow.svg" alt="down-arrow" />
+    <h3>Work</h3>
+</div>
+<h2 class="work">MY RECENT WORK</h2>
+<img src="image/Vector-4.png" alt="bar">
+<div class="placeholder">
+    <img src="image/placeholder.png" alt="blank-placeholder">
+</div>
+<div class="another-image">
+    <h3>Multi-Post Stories</h3>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+</div>
+<div>
+    <ul class="items">
+        <li>css</li>
+        <li>html</li>
+        <li>bootstrap</li>
+        <li class="ruby">Ruby</li>
+    </ul>
+</div>
+<button class="button" type="button">See Project</button>
```

Comment:
- [ ] I recommend you wrap each project into a `div` tag, 

```suggestion
<div>
<div class="placeholder">
    <img src="image/placeholder.png" alt="blank-placeholder">
</div>
<div class="another-image">
    <h3>Multi-Post Stories</h3>
        <p>
            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
        </p>
</div>
<div>
    <ul class="items">
        <li>css</li>
        <li>html</li>
        <li>bootstrap</li>
        <li class="ruby">Ruby</li>
    </ul>
</div>
<button class="button" type="button">See Project</button>
</div>
```

With this, you will have a more organic HTML